docs/higlass_server.rst

```diff
@@ -65,124 +65,75 @@ Additionally, the following settings are set via ``config.json`` (see ``config.j
-       "coordSystem":"mm10",
-       "coordSystem2":"",
-       "created":"2017-08-10T18:44:40.369924Z"
-    }
-
-API
----
-
-Retrieving a list of available tilesets:
+Retrieving properties of a tileset, for a specific `uuid`:
 
 .. code-block:: bash
 
-    curl localhost:8000/api/v1/tilesets
+  curl localhost:8000/api/v1/tilesets/${uuid}/
 
-To filter by a specific filetype, use the `t=filetype` parameter:
+To delete a tileset, specify the tileset `uuid` in the URL, and use the `DELETE` method with authentication credentials:
 
 .. code-block:: bash
 
-    curl localhost:8000/api/v1/tilesets?t=cooler
+  curl --user ${username}:${password} --request DELETE http://localhost:8000/api/v1/tilesets/${uuid}/
 
-To filter by datatype, use the `dt=datatype` parameter:
+To modify a tileset name, specify the tileset `uuid` in the URL, use the `PATCH` method with authentication credentials, and specify the new name in the JSON object passed to the request:
 
 .. code-block:: bash
 
-    curl localhost:8000/api/v1/tilesets?dt=matrix
+  curl --user ${username}:${password} --request PATCH --header "Content-Type: application/json" --data '{"name":"new_name_of_tileset"}' http://localhost:8000/api/v1/tilesets/${uuid}/
 
-Use the `dt` parameter to get gene annotations and chromsizes:
+Uploading data (POST)
+^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: bash
-
-    curl localhost:8000/api/v1/tilesets?dt=gene-annotation
-    curl localhost:8000/api/v1/tilesets?dt=chromsizes
-
-Retrieving properties of a tileset, for a specific `uuid`:
+The server API can be used to upload entire tilesets. To use this
+functionality, you need a username and password. These can be created using
+``higlass-manage create superuser``. The should be entered into a file (e.g.
+``~/.higlass-server-login)`` in the format ``username:password``.
 
- .. code-block:: bash
+The rest of the parameters should be specified according to the filetype,
+datatype, coordSystem and name of the dataset.
 
-    curl localhost:8000/api/v1/tilesets/${uuid}/
-
-To delete a tileset, specify the tileset `uuid` in the URL, and use the `DELETE` method with authentication credentials:
-
- .. code-block:: bash
-
-    curl --user ${username}:${password} --request DELETE http://localhost:8000/api/v1/tilesets/${uuid}/
-
-To modify a tileset name, specify the tileset `uuid` in the URL, use the `PATCH` method with authentication credentials, and specify the new name in the JSON object passed to the request:
-
- .. code-block:: bash
+.. code-block:: bash
 
-    curl --user ${username}:${password} --request PATCH --header "Content-Type: application/json" --data '{"name":"new_name_of_tileset"}' http://localhost:8000/api/v1/tilesets/${uuid}/
+  curl -u `cat ~/.higlass-server-login` \
+      -F "datafile=@/Users/peter/projects/negspy/negspy/data/mm10/chromInfo.txt" \
+      -F "filetype=chromsizes-tsv" \
+      -F "datatype=chromsizes" \
+      -F "coordSystem=mm10" \
+      -F "name=Chromosomes (mm10)" \
```

Comment:
FWIW: I would use single-quotes unless we know that we need interpolation. Bash can be "helpful" when you're not expecting it.